cfme/tests/physical_infrastructure/ui/test_physical_overview.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from cfme.physical.overview import PhysicalOverview
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3),
+    pytest.mark.provider([LenovoProvider])]
+
+
+def test_physical_overview_page():
+    view = navigate_to(PhysicalOverview, 'All')
+    assert view.is_displayed
+
+
+def test_physical_overview_servers_number(appliance, provider):
+    view = navigate_to(PhysicalOverview, 'All')
```

Comment:
Required:
Please register it in `entty_pioints` with a collection and then use this

Reply:
Along with the comment above, this navigation destination should be called 'Overview', and should be registered against the provider collection. Here, you can get `appliance.collections.physical_providers` to navigate with.